client/src/main/java/io/pravega/client/stream/EventWriterConfig.java

```diff
@@ -26,8 +26,8 @@
 
     public static final class EventWriterConfigBuilder {
         private int initalBackoffMillis = 1;
-        private int maxBackoffMillis = 60000;
-        private int retryAttempts = 6;
+        private int maxBackoffMillis = 20000;
+        private int retryAttempts = 10;
         private int backoffMultiple = 10;
```

Comment:
Since this configuration is for an active writer IMO we should reduce the max retry time to a lower value, maybe 10s. Also adjust the retries accordingly so we have a couple of minutes wait time - to allow for worst case recovery scenarios